sdk/typescript/README.md

```diff
@@ -40,6 +40,74 @@ const gatewayApi = GatewayApiClient.initialize({
 const { status, transaction, stream, state } = gatewayApi
 ```
 
+## Connecting to a secured gateway using authorization header
+
+This example shows how to provide basic auth, JWT and Bearer Token can be done the same way by passing the authorization header.
```

Comment:
This sentence doesn't work grammatically.
```suggestion
This example shows how to add an authorization header to a request, in order to make requests to a secured Gateway requiring basic auth, a JWT or a Bearer Token.
```

Reply:
fixed